### PR TITLE
Fix German translation No.3

### PR DESCRIPTION
**Pull request by erazortt**
Fix the German translation, Part 3 of 8
See also https://github.com/ihhub/fheroes2/issues/7860
And https://github.com/ihhub/fheroes2/pull/7865

### CODE REVIEW

File: files/lang/de.po [hunk 1]
```diff
@@ -2574,21 +2573,21 @@ msgid ""
...
+"Ihr müsst ein Zauberbuch kaufen, um die Magiergilde zu nutzen, aber Ihr habt "
+"derzeit keinen Platz für ein Zauberbuch. Versuchet, eines Eurer Artefakte "
+"einem anderen Helden zu übergeben."
 
 msgid "Exit"
 msgstr "Verlassen"
 
 msgid "Click to show next town."
-msgstr "Tippe, um die nächste Stadt anzuzeigen."
+msgstr "Clicken um die nächste Stadt anzuzeigen."
```
Comment (Laserlicht) [inline files/lang/de.po:2584]:
```suggestion
msgstr "Klicken um die nächste Stadt anzuzeigen."
```

File: files/lang/de.po [hunk 2]
```diff
@@ -2574,21 +2573,21 @@ msgid ""
...
 
 msgid "Show next town"
 msgstr "Nächste Stadt anzeigen"
 
 msgid "Click to show previous town."
-msgstr "Tippe, um die vorherige Stadt anzuzeigen."
+msgstr "Clicken um die vorherige Stadt anzuzeigen."
```
Comment (Laserlicht) [inline files/lang/de.po:2590]:
```suggestion
msgstr "Klicken um die vorherige Stadt anzuzeigen."
```

**Review by Laserlicht — CHANGES_REQUESTED** [review]

File: files/lang/de.po [hunk 3]
```diff
@@ -3103,16 +3103,20 @@ msgstr "%{skill} +1"
 msgid "You have learned %{skill}."
 msgstr "Ihr habt %{skill} gelernt."
 
-msgid "You may learn either:"
-msgstr "Ihr könnt lernen entweder:"
+#, fuzzy
```
Comment (Laserlicht) [inline files/lang/de.po:3106]: remove this?

Comment (erazortt) [reply files/lang/de.po]: yeah right, this and the next fuzzy can go. I just pushed an update.

File: files/lang/de.po [hunk 4]
```diff
@@ -3107,44 +3101,41 @@ msgid "%{skill} +1"
...
 msgid "You have learned %{skill}."
-msgstr "Sie haben %{skill} gelernt."
+msgstr "Ihr habt %{skill} gelernt."
 
-#, fuzzy
 msgid ""
 "%{name} has gained a level.\n"
 "\n"
 "%{skill} +1"
 msgstr "%{name} hat eine neue Stufe erreicht."
```
Comment (ihhub) [inline files/lang/de.po:3110]: Please translate the missing part as this string was recently modified. The full string is
```
"%{name} has gained a level.\n\n%{skill} +1"
```
Notice two **\n** symbols.

File: files/lang/de.po [hunk 5]
```diff
@@ -3107,44 +3101,41 @@ msgid "%{skill} +1"
...
 
-#, fuzzy
 msgid ""
 "You may learn either:\n"
 "%{skill1}\n"
 "or\n"
 "%{skill2}"
-msgstr "Sie dürfen lernen entweder:"
+msgstr "Ihr könnt lernen entweder:"
```
Comment (ihhub) [inline files/lang/de.po:3117]: Please translate the missing part as this string was recently modified. The full string is
```
"You may learn either:\n%{skill1}\nor\n%{skill2}"
```
Notice three **\n** symbols and also **or** in between them.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @erazortt , I left 2 comments here. Would you mind to take a look at them?

File: files/lang/de.po (hunk 5 above)
Comment (erazortt) [reply files/lang/de.po]: ok done

File: files/lang/de.po [hunk 6]
```diff
@@ -3643,19 +3637,19 @@ msgid "Save Map"
...
 msgid "Monster"
 msgstr "Monster"
 
 msgid "Monsters cannot be placed on water."
-msgstr ""
+msgstr "Monster können nicht auf Wasser plaziert werden."
 
 msgid "Choose a tile which does not contain any objects."
-msgstr ""
+msgstr "Ein Feld wählen, das keine Objekt enthält."
```
Comment (erazortt) [inline files/lang/de.po:3652]: I also just translated this new string and the one just above, which both were just added by another commit to master.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@erazortt , thanks a lot for this update!

**Pull request merged**